packages/corvid-local-site/src/codeCompletion.js

```diff
@@ -0,0 +1,45 @@
+const map_ = require("lodash/map");
+const logger = require("corvid-local-logger");
+const {
+  pageTsConfigFilePath,
+  masterPageTsConfigFilePath,
+  backendTsConfigFilePath,
+  publicTsConfigFilePath
+} = require("./sitePaths");
+
+let corvidTypes;
+try {
+  corvidTypes = require("corvid-types");
+} catch (e) {
+  logger.warn("Using corvid-cli without code completion");
```

Comment:
why do you consider it a warning ? (it'll get to sentry)

Reply:
I wasn't aware that it will get to sentry, should i change it to logger.info?

Reply:
@dor-itzhaki ?

Reply:
yes